### PR TITLE
Added blockboy/ac980mini keyboard folder

### PR DESCRIPTION
**Pull request by rooski15**
## Description

Built keyboard folder to support the Block Boy "AC980Mini" keyboard.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/blockboy/ac980mini/readme.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# blockboy/ac980mini
+
+![AC980mini](https://ae01.alicdn.com/kf/H2955001a66544ee1ac38a42b0632fe41m.png)
+![AC980mini kle](https://imgur.com/prgMVo2.jpg)
```
Comment (drashna) [inline keyboards/blockboy/ac980mini/readme.md:4]:
```suggestion
![AC980mini kle](https://imgur.com/prgMVo2h.jpg)

```

File: keyboards/blockboy/ac980mini/readme.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# blockboy/ac980mini
+
+![AC980mini](https://ae01.alicdn.com/kf/H2955001a66544ee1ac38a42b0632fe41m.png)
```
Comment (drashna) [inline keyboards/blockboy/ac980mini/readme.md:3]: Please rehost this image on an image hosting site such as imgur.

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/blockboy/ac980mini/readme.md (hunk 1 above)
Comment (rooski15) [reply keyboards/blockboy/ac980mini/readme.md]: Done!

File: keyboards/blockboy/ac980mini/ac980mini.c [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+/*
+void suspend_power_down_kb(void) {
+    rgb_matrix_set_suspend_state(true);
+    suspend_power_down_user();
+}
+
+void suspend_wakeup_init_kb(void) {
+    rgb_matrix_set_suspend_state(false);
+    suspend_wakeup_init_user();
+}
```
Comment (drashna) [inline keyboards/blockboy/ac980mini/ac980mini.c:60]: This is deprecated and should be removed, regardless if it's actively being used or not. 
```suggestion
```

File: keyboards/blockboy/ac980mini/rules.mk [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no        # Commands for debug and configuration
+NKRO_ENABLE = no            # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
```
Comment (drashna) [inline keyboards/blockboy/ac980mini/rules.mk:19]:
```suggestion
COMMAND_ENABLE = no         # Commands for debug and configuration
NKRO_ENABLE = no            # Enable N-Key Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/blockboy/ac980mini/ac980mini.c [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+// Enable to manually toggle Num / Caps
+__attribute__ ((weak))
+void rgb_matrix_indicators_user(void) {
+    if (host_keyboard_led_state().caps_lock) {
+        rgb_matrix_set_color(13, 0xFF, 0xFF, 0xFF);
+    }
+	if (host_keyboard_led_state().num_lock) {
+        rgb_matrix_set_color(34, 0xFF, 0xFF, 0xFF);
+   }
+}
```
Comment (drashna) [inline keyboards/blockboy/ac980mini/ac980mini.c:62]: Why bother including this then?  You'd be better off throwing this into the keymap.c file, in that case. 
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/blockboy/ac980mini/ac980mini.c (hunk 5 above)
Comment (rooski15) [reply keyboards/blockboy/ac980mini/ac980mini.c]: My misunderstanding. This file was initially copied from another, recently commited keyboard folder, so I was unaware this was an issue.

**Review by drashna — APPROVED** [review]

**Comment by rooski15** [comment]
Just checking if there's any thing else I need to do here?

**Comment by drashna** [comment]
Mostly, just waiting on a second review.

File: keyboards/blockboy/ac980mini/readme.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
+# blockboy/ac980mini
```
Comment (fauxpark) [inline keyboards/blockboy/ac980mini/readme.md:1]: Please use Unix line endings for all files.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/blockboy/ac980mini/readme.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+![AC980mini](https://i.imgur.com/1ZAROm2h.png)
+![AC980mini kle](https://imgur.com/prgMVo2h.jpg)
+
+
+*An F-Row-Less 1800 project with fixed layout and lots of RGB.*
```
Comment (fauxpark) [inline keyboards/blockboy/ac980mini/readme.md:7]:
```suggestion
An F-Row-Less 1800 project with fixed layout and lots of RGB.
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Comment by rooski15** [comment]
Checking back in. Is there more I need to change here?

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**